src/chain/index.ts

```diff
@@ -23,12 +25,14 @@ export class BeaconChain extends EventEmitter {
   private db: DB;
   private eth1: Eth1Notifier;
   private _latestBlock: BeaconBlock;
+  private forkChoice: LMDGHOST;
```

Comment:
Would be nice to set interface type here in case fork choice implementation changes

Reply:
:+1:  will make an issue for this